entities/src/main/java/com/flipkart/varadhi/entities/SubscriptionShard.java

```diff
@@ -1,9 +1,10 @@
 package com.flipkart.varadhi.entities;
 
 import lombok.AllArgsConstructor;
+import lombok.Getter;
 import lombok.Value;
 
-
+@Getter
```

Comment:
jackson bean serialization needs Getter or Setter for figuring out the properties. Without this it was not serializable